packages/x-date-pickers/src/tests/fieldKeyboardInteraction.test.tsx

```diff
@@ -39,8 +43,8 @@ const adapterToTest = [
   'dayjs',
   'moment',
   'date-fns-jalali',
-  'moment-hijri',
-  'moment-jalaali',
+  // 'moment-hijri',
+  // 'moment-jalaali',
```

Comment:
In my browser, if I render an `AdapterMomentJalaali` date, the year is a number but in the test it's some character (from the persian alphabet I suppose) which don't work at all with our editing (since we are doing `Number(section.value)`).

I tried to understand why and failed...
If you want to have a look, for me everything works well on my browser but the tests behavior make no sense...

Reply:
I've pushed [a fix for `moment-jalaali`](https://github.com/mui/mui-x/pull/7993/commits/7fd66494ca1d9ab913c700146d471d3edfad3050). Let me know if you think that it's an inappropriate one. 🙈 
As for `moment-hijri`, we are well aware that it is broken and have listed it in our doc. 🤔

Reply:
:partying_face: 

We just have a small issue where `moment` keeps using the persian on the rest of its test

Reply:
I'll reset it in an `after`